File: Day-10/Pattern11.py
n = int(input())

def PatternEleven(n):
    i = 1
    ch = 65
    while(i <= n):
        j =1
        while(j <=n):
            print(chr(ch), end='')
            j+= 1
        i +=1
        ch += 1
        print(end='\n')
    
PatternEleven(n)

File: Day-10/Pattern11.js
function PatternEleven(n) {
  let i = 1;
  let ch = 65;
  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= n) {
      str += String.fromCharCode(ch);
      j++;
    }
    i++;
    ch++;
    console.log(str);
  }
}

PatternEleven(3);
